src/main/java/org/folio/db/model/NoteTypeView.java

```diff
@@ -0,0 +1,61 @@
+package org.folio.db.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import org.folio.rest.jaxrs.model.Link;
+import org.folio.rest.jaxrs.model.Metadata;
+import org.folio.rest.jaxrs.model.NoteTypeUsage;
+import org.folio.rest.jaxrs.model.UserDisplayInfo;
+
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Database representation of a note
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class NoteTypeView {
```

Comment:
why do you need another note type object if there is NoteType already which has the same content

Reply:
Thank you. It makes sense to refactor this issue.